### DISCUSSION TITLE
各タイムラインに流れてくる内容についての仕様確認

### DISCUSSION THREAD

**Discussion by samunohito** [misskey-dev/misskey#12318] [Q&A] [answered]
MisskeyHubの[タイムライン](https://misskey-hub.net/docs/features/timeline.html)ページにタイムラインの説明が記載されておりますが、チャンネルを含めた場合はどのようになるのか分からなかったので仕様確認をさせて頂きたいです。
上記ページの記載内容にチャンネルを含めると以下表のような形になると認識しているのですが、相違はございますでしょうか？

|ソース|配信先タイムライン|||||
|:----|:----|:----:|:----:|:----:|:----:|
|ユーザーのフォロー状況|公開範囲|ホーム|ローカル|ソーシャル|グローバル|
|フォロー有(ローカル)|公開|✔|✔|✔|✔|
||ホーム|✔||✔||
||フォロワー|✔||✔||
||ダイレクト|✔||✔||
||チャンネル(フォロー有)|✔||✔||
||チャンネル(フォロー無)|||||
|フォロー有(リモート)|公開|✔||✔|✔|
||ホーム|✔||✔||
||フォロワー|✔||✔||
||ダイレクト|✔||✔||
||チャンネル(フォロー有)|||||
||チャンネル(フォロー無)|||||
|フォロー無(ローカル)|公開||✔|✔|✔|
||ホーム|||||
||フォロワー|||||
||ダイレクト|||||
||チャンネル(フォロー有)|✔||✔||
||チャンネル(フォロー無)|||||
|フォロー無(リモート)|公開||||✔|
||ホーム|||||
||フォロワー|||||
||ダイレクト|||||
||チャンネル(フォロー有)|||||
||チャンネル(フォロー無)|||||

**Comment by syuilo** [comment, marked as answer]
合っってそう

**Reply by samunohito** [reply]
ありがとうございます:pray:
